### DISCUSSION TITLE
Check chunk options

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#2678] [Feature Requests]
Recently, it took me longer than I'd like to admit to find out why the following code chunk gets evaluated despite setting the corresponding option:

````
---
title: Test
---

```{r}
#| eval: false
cat("Hello")
```
````

If you want to reproduce, copy this example to a file `test.qmd` and preview it with `quarto preview test.qmd`.

In fact, the cell is evaluated because I accidentally used a non-breaking space (U+00A0, <kbd>⌥</kbd><kbd>Space</kbd> on a Mac) instead of a normal space after the `#|` characters. I now turned on `"editor.unicodeHighlight.includeComments": true` and `"editor.unicodeHighlight.nonBasicASCII": true` in Visual Studio Code, which makes this obvious, but I thought that maybe Quarto could issue a warning (or info) when encountering unknown chunk options. Currently, I didn't find anything even after setting `--log-level info`. WDYT?

**Comment by cderv** [comment]
Thanks for the report. Let me transform that to an issue to discuss how to improve this, as there is several possibility IMO.

**Comment by mcanouil** [comment]
This was completed in #2681

**Reply by cbrnr** [reply]
Great, thanks a lot!
